recipes/py-dom-xpath-six/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "py-dom-xpath-six" %}
+{% set version = "0.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d6c319f23ee6e71d415f3a588f6d245495c5d41cb2d2a35bd80777c4a6d2e50b
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
```

Comment:
Since this is a `noarch: python` package, you could change this to `python =2.7|>=3.6` so that people still on python 2.7 can install it, since officially the package still supports it. If so also change the corresponding below in `run`.

Reply:
Thank you for the suggestion. Python version 2.7 is included in the recent commit